php-practice_1.php

```diff
@@ -1,41 +1,139 @@
 <?php
 // Q1 変数と文字列
+$name = '友田';
 
+$myname = '私の名前は「' . $name . '」です。';
+var_dump($myname);
 
 // Q2 四則演算
-
+$x = 5;
+$y = 4;
+$num = $x * $y;
+var_dump($num);
+var_dump($num / 2);
 
 // Q3 日付操作
-
+$timezone = new DateTimeZone('Asia/Tokyo');
+$now = new DateTime('now', $timezone);
+$time = '現在時刻は、' . $now->format('Y年m月d日 H時i分s秒') . 'です。';
+var_dump($time);
 
 // Q4 条件分岐-1 if文
-
+$$device = 'windows';
+if ($device === 'windows' || $device === 'mac') {
+  echo '使用OSは、' . $device . 'です。';
+} else{
+  echo 'どちらでもありません。';
+}
 
 // Q5 条件分岐-2 三項演算子
-
+$age = 21;
+$message = ($age >= 18) ? '成人です。' : '未成年です。';
+echo $message;
 
 // Q6 配列
-
+$kanto = ['東京都', '茨城県', '群馬県', '栃木県', '千葉県','埼玉県' , '神奈川県'];
+echo $kanto[3] . 'と' . $kanto[4] . 'は関東地方の都道府県です。';
 
 // Q7 連想配列-1
-
+$array = [
+  '東京都'=>'新宿区',
```

Comment:
連想配列の`=>`前後は半角スペースを設けてください！
```suggestion
  '東京都' => '新宿区',
```

Reply:
`=>`の前後に半角スペースを入れて修正しました。